build/Common.props

```diff
@@ -7,6 +7,7 @@
     <DefineConstants>$(DefineConstants);SIGNED</DefineConstants>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <NetFrameworkMinimumSupportedVersion>net462</NetFrameworkMinimumSupportedVersion>
+    <AnalysisLevel>latest-All</AnalysisLevel>
```

Comment:
latest-All sound a bit too much, can we use a fixed version here?

Reply:
There is an agreement to use this approach on SDK repository. I think we could do the same here. See https://github.com/open-telemetry/opentelemetry-dotnet/issues/3958

Reply:
Makes sense.